src/lib/containers/download_list/DownloadListTable.tsx

```diff
@@ -227,17 +345,105 @@ export default function DownloadListTable(props: DownloadListTableProps) {
         <ReactBootstrap.Table striped={true} responsive={true}>
           <thead>
             <tr>
-              <th>File Name</th>
+              <th>
+                File
+                <button
+                  className={`sort SRC-primary-background-color-hover ${
+                    sortedColumn.column === 'file'
+                      ? 'SRC-primary-background-color'
+                      : ''
+                  }`}
+                  onClick={() => {
+                    sortColumn('file')
+                  }}
+                >
+                  <FontAwesomeIcon
+                    icon={
+                      sortedColumn.column === 'file'
+                        ? sortedColumn.isDescending === false
+                          ? faSortAmountDown
+                          : faSortAmountUp
+                        : faSortAmountDown
+                    }
+                  />
+                </button>
+              </th>
               <th>Access</th>
-              <th>Created By</th>
-              <th>Created On</th>
-              <th>Size</th>
+              <th>
+                Created By
+                <button
+                  className={`sort SRC-primary-background-color-hover ${
+                    sortedColumn.column === 'createdBy'
+                      ? 'SRC-primary-background-color'
+                      : ''
+                  }`}
+                  onClick={() => {
+                    sortColumn('createdBy')
+                  }}
+                >
+                  <FontAwesomeIcon
```

Comment:
we need to set the color of the icon to white if the column has been clicked, the logic for the icon state can be simplified by getting rid of the nested ternary statement
e.g.
```jsx
  <FontAwesomeIcon
      icon={
        sortedColumn.column === 'size'
          && sortedColumn.isDescending === true ?
           : faSortAmountUp
           : faSortAmountDown
      }
      color={sortedColumn.column === 'size' ? 'white' : ''}
      />
```